### DISCUSSION TITLE
Compiling Firefox with cosmo

### DISCUSSION THREAD

**Discussion by betim** [jart/cosmopolitan#1328] [Q&A]
Is it possible to compile Firefox with cosmocc instead of GCC?

**Comment by jart** [comment]
I don't know, because no one's tried yet. I would love for that to be possible. It may be difficult, because Firefox needs to depend on a lot of platform-specific dynamic shared objects. Integrating with each one of those usually poses a unique set of challenges. See cosmoaudio in this repo for an example.

**Reply by betim** [reply]
Thank you for your answer.

I believe closest person to succeed doing this is you and getting it to work on Linux would be easiest (I think) and most desirable. I'd try doing this, but am not very experienced.

Cheers
